node/cn/channel_manager_test.go

```diff
@@ -55,6 +55,10 @@ func testChannelManager(t *testing.T, chSize int) {
 		// Before calling RegisterChannelWithIndex,
 		// calling GetChannelWithMsgCode with registered MsgCode should return no channel and no error.
 		for i := StatusMsg; i < MsgCodeEnd; i++ {
+			if i == DummyMsg1 || i == DummyMsg2 {
```

Comment:
Where is the definition of `DummyMsg1` ?

Reply:
maybe Unused10? 
https://github.com/klaytn/klaytn/pull/1394/files?w=0#diff-102c0c38488b22c8d83cc296c1d1bd21330d33fdf8ba3483c2464c0883fec602R81